src/types/onyx/Policy.ts

```diff
@@ -142,7 +142,7 @@ type QBOConnectionData = {
     bankAccounts: Account[];
     creditCards: Account[];
     accountsReceivable: Account[];
-    accountsPayable: Account[];
+    accountPayable: Account[];
```

Comment:
I saw this change at a few places with `accountsPayable` replaced with `accountPayable`, but considering it's an array should we be retaining plural form?

Reply:
Pretty sure `accountPayable` is how we get it from the QBO connection data. The `accountsPayable` typo we have now in prod on NewDot causes the list of AP accounts to be blank.

Reply:
Thanks for confirming. Just about to test it now.

Reply:
We can't rename this fields at this point otherwise the backend won't recognize them. We have to stick with `accountPayable` even if it is an array